content/en/docs/setup/production-environment/tools/kubespray.md

```diff
@@ -31,11 +31,11 @@ To choose a tool which best fits your use case, read [this comparison](https://g
 
 ### (1/5) Meet the underlay requirements
 
-Provision servers with the following [requirements](https://github.com/kubernetes-incubator/kubespray#requirements):
+Provision servers with the following [requirements](https://github.com/kubernetes-sigs/kubespray#requirements):
 
-* **Ansible v2.5 (or newer) and python-netaddr is installed on the machine that will run Ansible commands**
+* **Ansible v2.7.8 and python-netaddr is installed on the machine that will run Ansible commands**
 * **Jinja 2.9 (or newer) is required to run the Ansible Playbooks**
-* The target servers must have **access to the Internet** in order to pull docker images
+* The target servers must have access to the Internet in order to pull docker images. Otherwise, additional configuration is required ([See Offline Environment](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/downloads.md#offline-environment))
```

Comment:
nit:
```suggestion
* The target servers must have access to the Internet in order to pull docker images. Otherwise, additional configuration is required--see [Offline Environment](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/downloads.md#offline-environment).
```